Cheers.java

```diff
@@ -1,6 +1,28 @@
 // Prints a crowd cheering output.
 public class Cheers {
         public static void main(String[] args) {
-	    // Replace this comment with your code
+	    String str=args[0];
+            int n=Integer.parseInt(args[1]);
+            String letters = "AEFHILMNORSX";
+            String letters_small="aefhilmnorsx";
+            String line= "";
+            for (int i = 0; i <str.length(); i++)
+            {
+                if ((letters.indexOf(str.charAt(i))==-1)&&(letters_small.indexOf(str.charAt(i))==-1)) {
+                        line="Give me a  "+ str.toUpperCase().charAt(i)+": "+str.toUpperCase().charAt(i)+"!";
+                        System.out.println(line);
+
+                }
+                else{
+                        line= "Give me an " + str.toUpperCase().charAt(i)+": "+str.toUpperCase().charAt(i)+"!";
```

Comment:
Too many toUpperCase() calls.. can do it in first line and will save a lot of trouble